packages/components/src/ListView/Items/Item/Item.component.js

```diff
@@ -8,58 +8,71 @@ function itemLabelClasses() {
 	return classNames(theme['tc-listview-item-label'], 'tc-listview-item-label');
 }
 
-function Item(props) {
-	const { id, item, parentItem, isSwitchBox, searchCriteria } = props;
-
-	/**
-	 * This function allow to get component rendering based on searchCriteria
-	 * @param label the current label to parse & to render
-	 */
-	function getSearchedLabel(label) {
-		const splitRegex = new RegExp(`(${searchCriteria})`, 'gi');
-		return label.split(splitRegex).map((part, index) => {
-			const higlighted = part.toLowerCase() === searchCriteria.toLowerCase() ? <b>{part}</b> : part;
-			return <span key={index}>{higlighted}</span>;
-		});
+class Item extends Component {
+	componentWillReceiveProps(nextProps) {
```

Comment:
Please avoid using componentWillReceiveProps (will be deprecated in future versions of React).
Instead use `componentDidUpdate(prevProps)`. You then compare prevProps with current props to call `measure()`